test/verify/check-networking-firewall

```diff
@@ -191,6 +191,7 @@ class TestFirewall(NetworkCase):
             addService('public', 'freeipa-4')
             b.click(".zone-section[data-id='public'] tr[data-row-id='freeipa-4']")
             b.wait_present(".zone-section[data-id='public'] tbody.open .listing-ct-body:contains(Included Services)")
+            b.click(".zone-section[data-id='public'] tr[data-row-id='freeipa-4']")
```

Comment:
What does that extra click do? The one two lines above looks like opening the expander, and that new one closes it again?

Reply:
Later they're removed in a loop, where the expander is opened again.